tasks/main.yml

```
@@ -153,11 +153,21 @@
   tags:
     - rbenv
 
+- name: check if current system ruby version is {{ rbenv.ruby_version }}
+  shell: $SHELL -lc "rbenv version | cut -d ' ' -f 1 | grep -Fx '{{ rbenv.ruby_version }}'"
+  register: ruby_selected
+  changed_when: false
+  ignore_errors: yes
```

You want check success of ruby installation? Should we ignoring this error?